backups/Backup-Infrastructure.Tests.ps1

```diff
@@ -0,0 +1,163 @@
+. $PSScriptRoot\Backup-Infrastructure.ps1
+
+# NOTE: Functions below are propertly defined only when Veeam snapin is loaded.
+#       For tests, these dummy definitions are sufficient to enable mocking.
+function Connect-VBRServer {}
+function Find-VBRViEntity { Param($Name) return $Name }
+function Start-VBRZip {
+    Param(
+        [string]$Folder,
+        [string]$Entity,
+        [string]$Compression,
+        [switch]$DisableQuiesce)
+
+    $path = Join-Path $Folder $Entity
+    $path = $path + ".vbk"
+    New-Item $path -ItemType File
+}
+function Disconnect-VBRServer {}
+
+Describe 'Backup-Infrastructure' -Tags 'CI' {
```

Comment:
Consider adding `Unit` tag to both test suites, so that `./Invoke-Selfcheck.ps1` detects & runs them. Also we should verify if it works when ran using `./Invoke-Selfcheck.ps1`

Reply:
Thanks, I'll add the tags. Also `Invoke-Selfcheck.ps1` is picking it up right now: http://logs.opencontrail.org/winci/github/WinContrail/contrail-win-ci-gh/1935/TestReports/CISelfcheck/pretty_test_report/Index.html
Only needed to add `CI` tag and stash the code in Jenkinsfile.